README.md

```diff
@@ -0,0 +1,20 @@
+Current functions:
+StartMetrics() - starts the metrics server over an endpoint and port. Should be configurable with a sane default
+
+RegisterMetrics() - Registers all metrics that are given to metricsList with the metrics server. Need a way to pass a metricsList or to register individual metrics easily.
```

Comment:
This seems to be a WIP description. Please only specify what the function does, and create issues if you want to spend time changing behavior.

Reply:
See above